st2client/st2client/shell.py

```diff
@@ -241,6 +242,10 @@ def get_client(self, args, debug=False):
             try:
                 token = self._get_auth_token(client=client, username=username, password=password,
                                              cache_token=cache_token)
+            except requests.exceptions.ConnectionError as e:
+                LOG.warn('API server is not available, skipping authentication.')
+                LOG.exception(e)
+                return client
```

Comment:
That said, I am curious how would client.* operations would work if auth server died but auth is enabled in server. We'd keep getting 401s back?